employee-rostering-frontend/src/index.tsx

```diff
@@ -29,7 +29,7 @@ import { I18nextProvider } from 'react-i18next';
 import { configureStore } from 'store';
 
 const store = configureStore({
-  restBaseURL: '/rest'
+  restBaseURL: `${process.env.REACT_APP_BACKEND_URL}/rest`
```

Comment:
+1 This is useful during development. I use this approach in VRP so that I don't have to use proxy feature of the development server, and let frontend talk directly to backend. I can then start backend on different port and change REACT_APP_BACKEND_URL in `.env.development.local` without having to touch package.json where the development server proxy can be configured.